packages/gatsby/cache-dir/production-app.js

```diff
@@ -12,6 +12,7 @@ import history from "./history"
 window.___history = history
 import emitter from "./emitter"
 window.___emitter = emitter
+import { isEqual } from "lodash"
```

Comment:
Please use the shallow-compare package as we're already using that so it require any extra code https://github.com/gatsbyjs/gatsby/blob/3ab063feb7997d1c03809cd5ff4d754522df1ac7/packages/gatsby/cache-dir/component-renderer.js#L6